Frontend/src/app/components/research-lab/signal-report/signal-verdict-block/signal-verdict-block.component.html

```
@@ -0,0 +1,72 @@
+@let g = graduation();
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)